EcoSoapBank/Model/ImpactStats.swift

```diff
@@ -15,4 +15,18 @@ struct ImpactStats: Decodable {
     let paperRecycled: Int?
     let peopleServed: Int?
     let womenEmployed: Int?
+
+    init(soapRecycled: Int? = nil,
+         bottlesRecycled: Int? = nil,
+         linensRecycled: Int? = nil,
+         paperRecycled: Int? = nil,
+         peopleServed: Int? = nil,
+         womenEmployed: Int? = nil) {
+        self.soapRecycled = soapRecycled
+        self.bottlesRecycled = bottlesRecycled
+        self.linensRecycled = linensRecycled
+        self.paperRecycled = paperRecycled
+        self.peopleServed = peopleServed
+        self.womenEmployed = womenEmployed
```

Comment:
Is there are reason for adding the initializer here instead of just using the default member-wise init? (same in property and user)

Reply:
Actually there isn't now. I had added that when Jon suggested it to be able to put break points in, but now that I've figured out the issue, I could delete these. I just wasn't thinking well enough about that yesterday. I'll fix that and push it up here before I merge it.